### ISSUE TITLE
[Flaw] Logic in Developer Prompt

### ISSUE THREAD

**Issue by bogoconic1** [bogoconic1/Qgentic-AI#34]
In the prompt, the approach was to ask for a script that ```sets an arbitrarily high number of epochs for training and perform early stopping```

This can be a problem if the AI chooses to implement ```warmup``` using a normal learning rate - early epochs learn slower and hence the model will take longer to converge, leading to longer script running times.
